doc/reference/tooling/tcm/tcm_releases/tcm_1.1.rst

```diff
@@ -0,0 +1,117 @@
+.. _tcm_releases_1_1:
+
+Tarantool Cluster Manager 1.1
+=============================
+
+Release date: May 16, 2024
+
+Releases in series: 1.1.0
+
+|tcm_full_name| 1.1 introduces a number of new features that extend and improve its
+cluster management capabilities. Below is an overview of its key updates.
+
+.. _tcm_releases_1_1_data_access:
+
+Data access
+-----------
+
+An important update of |tcm| 1.1.0 is a set of features that enable access to clusters'
+stored data.
+
+The instance *space explorer* shows all spaces that exist on an instance, including
+system spaces. On its pages, you can view and edit the stored data. To open the instance explorer,
+find the instance on the cluster stateboard and click its name to open its details page.
+Then click **Explorer** in the **Actions** menu in the top right corner.
+
+In the development mode, the instance explorer also includes the *schema editor*.
+It allows you to add new and edit existing spaces.
+
+For clusters that use the `CRUD <https://github.com/tarantool/crud>`__ module,
+there is also the *CRUD explorer* that enables access to data in user spaces across
+the entire cluster on one page. The CRUD explorer is located on the **Tuples** page.
```

Comment:
> access to data in user spaces across the entire cluster on one page

Technically speaking, `on one page` sounds like all the data can be show on one page at one - without any pagination :) Maybe, we can drop this to avoid confusion.